oop_verify.py

```diff
@@ -124,14 +124,9 @@ class ProjectFilesVerifier(Verifier):
     def __init__(self):
         Verifier.__init__(self)
         self.pages = []
-        self.minimum_size = 410
+        self.minimum_size = 380
         self.page_elements = [
```

Comment:
@sf2ne Are you referring to the size? Page elements refers to a list of actual HTML elements used for spot check.